azurerm/internal/services/policy/policy_assignment_resource.go

```diff
@@ -154,6 +188,8 @@ func resourceArmPolicyAssignmentCreateUpdate(d *schema.ResourceData, meta interf
 			DisplayName:        utils.String(d.Get("display_name").(string)),
 			Scope:              utils.String(scope),
 			EnforcementMode:    convertEnforcementMode(d.Get("enforcement_mode").(bool)),
+			Metadata:           nil,
+
```

Comment:
Could we remove these two line? In a struct initialization, by neglecting one attribute is equivalent to assigning it to `nil`

Reply:
done